src/app/client/src/app/modules/resource/components/collection-player/collection-player.component.ts

```diff
@@ -141,8 +145,8 @@ export class CollectionPlayerComponent implements OnInit, OnDestroy {
       this.objectContentInteract = {
         id: content.metadata.identifier,
         type: content.metadata.contentType || content.metadata.resourceType || 'content',
-        ver: content.metadata.pkgVersion ? content.metadata.pkgVersion.toString() : '1.0',
-        rollup: {l1: this.collectionId}
+        ver: content.metadata.pkgVersion || '1',
```

Comment:
Same as above comment: why is this changed? previously, we were using "1.0" as the default version number.

Reply:
I had merge conflicts, so i took the latest, these changes are theirs